src/types/models/listing.ts

```diff
@@ -19,6 +19,30 @@ export interface DealerListingImages {
   spinCode: string
 }
 
+export interface DealerDefaultAdditionalServices {
+  deliveryFeeIncluded: boolean
+  expertInstructionIncluded: boolean
+  fullTankIncluded: boolean
+  nextInspectionIncluded: boolean
+  otherServices: string
+  vignetteIncluded: boolean
+}
+
+export interface DealerDefaultWarranty {
+  warrantyDetails: string
+  warrantyDuration: number
+  warrantyMileage: number
+  warrantyStartDate: string
+  warrantyType: string
+}
+
+export interface DealerDefaultListing
+  extends DealerDefaultAdditionalServices,
+    DealerDefaultDescription,
+    DealerDefaultWarranty {
+  description: string
```

Comment:
wouldn't `DealerDefaultDescription` cover that already?

Reply:
i removed the DealerDefaultDescription but it doesnt appear in github. maybe they have issues.

i did ```
export interface DealerDefaultListing
  extends DealerDefaultAdditionalServices,
    DealerDefaultWarranty {
  description: string
}
```

Reply:
Oh, you probably did that mid-review. I think it makes sense to keep it for the sake of consistency.